sensor_msgs/CompressedImage foveated_image
attention_package/Tuple[] bounding_box_origins
attention_package/Tuple[] bounding_box_ends
